### DISCUSSION TITLE
Github Star Count is incorrect

### DISCUSSION THREAD

**Discussion by alias-rahil** [community/community#22042] [New to GitHub]
<p>I have just removed all stars(Unstared) from Github, but the count remains 3.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/3/637123aebee84daafcede2643ddc756ed8b698f3.jpeg" data-download-href="/uploads/short-url/ebHFvTRjgpk0WELZ8IH1bBArdrJ.jpeg?dl=1" title="Screenshot_2021-08-05_01-54-28" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59060219/181068447-497470e1-b337-4c7f-9d00-1f5687fa301b.jpeg" alt="Screenshot_2021-08-05_01-54-28" data-base62-sha1="ebHFvTRjgpk0WELZ8IH1bBArdrJ" width="690" height="378" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/3/637123aebee84daafcede2643ddc756ed8b698f3_2_690x378.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/6/3/637123aebee84daafcede2643ddc756ed8b698f3_2_1035x567.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/3/637123aebee84daafcede2643ddc756ed8b698f3_2_1380x756.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/3/637123aebee84daafcede2643ddc756ed8b698f3_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_2021-08-05_01-54-28</span><span class="informations">1920×1053 126 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jsoref** [comment, marked as answer]
<p>I’d file a bug. While star counts don’t appear to update live, I don’t think I’d expect them to be stale for over a day.</p>
<p><a href="https://support.github.com/" class="onebox" target="_blank" rel="noopener">https://support.github.com/</a></p>
